modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/internal/search/spi/model/index/contributor/test/JournalArticleModelDocumentContributorTest.java

```diff
@@ -129,6 +129,9 @@ private String _getDDMIndexerContent(String languageId) throws Exception {
 	private Document _getDocument() {
 		DocumentImpl documentImpl = new DocumentImpl();
 
+		Assert.assertNotNull(_journalArticle.getDDMFormValues());
+		Assert.assertNotNull(_journalArticle.getDDMFormValues());
```

Comment:
Why checking it twice? Is this on purpose?

Reply:
My mistake. I've just sent again fixing it. 

thanks @adolfopa